tests/connector/tests/hydrator_api_test.go

```diff
@@ -31,25 +23,30 @@ func TestHydrators(t *testing.T) {
 	require.NotEmpty(t, app.ID)
 	appID := app.ID
 
+	runtime, err := fixtures.RegisterRuntimeFromInputWithinTenant(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, &graphql.RuntimeInput{
+		Name: "test-hydrators-runtime",
+	})
+	defer fixtures.CleanupRuntime(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, &runtime)
+	require.NoError(t, err)
+	require.NotEmpty(t, runtime.ID)
+	runtimeID := runtime.ID
+
 	hash := "df6ab69b34100a1808ddc6211010fa289518f14606d0c8eaa03a0f53ecba578a"
 
 	for _, testCase := range []struct {
 		clientType           string
-		clientId             string
```

Comment:
Having unified `cliet_id`, removes the need of` if testCase.clientType == "Application" {` everywhere except in the check here: https://github.com/kyma-incubator/compass/pull/2007/files#diff-c5b72ac3ff812b64b53e2ea366f61ea7068307ff8b5e7e8ce07b5e19e4de84daR104 , so I would propose to return it.